blocks/init/src/blocks/custom/column/components/column-responsive-tab-content.js

```diff
@@ -36,7 +36,8 @@ export const ColumnResponsiveTabContent = (props) => {
               {__('Width', 'eightshift-boilerplate')}
             </Fragment>
           }
-          help={sprintf(__('Option to change width block in grid from the left. Change block width in %d columns range. Example 6 = 50% screen width. If you set a value to -1 it will not be used and the parent brakepoint will be used.', 'eightshift-boilerplate'), globalSettings.maxCols)}
+          help={sprintf(__('Option to change width block in grid from the left. Change column width in %d columns range. Example 6 = 50 percent of the screen width. If you set a value to -1 it will not be used and the parent brakepoint will be used.', 'eightshift-boilerplate'), globalSettings.globalVariables.maxCols)}
```

Comment:
```suggestion
          help={sprintf(__('Option to change the block width in the grid from the left. Change column width in %d columns range. Example: 6 is 50% of the screen width. If you set a value to -1 it will not be used and the parent breakpoint will be used.', 'eightshift-boilerplate'), globalSettings.globalVariables.maxCols)}
```